Improve I18N Issues

alexclassroom opened the pull request:
- Unified UI string writing
- Punctuation should be included in translatable UI strings
- Added translatable UI strings
- For security reason, change `_e` to `esc_html_e`

alexclassroom committed:
Improve I18N Issues

- Unified UI string writing
- Punctuation should be included in translatable UI strings
- Added translatable UI strings
- For security reason, change `_e` to `esc_html_e`

toolstack commented:
@alexclassroom thanks for the PR, but there doesn't seem to be any changes to any files...

alexclassroom committed:
Improve I18N Issues

- Unified UI string writing
- Punctuation should be included in translatable UI strings
- Added translatable UI strings
- For security reason, change `_e` to `esc_html_e`
- Sorry, this is my fault, I forgot to upload the updated files.

alexclassroom commented:
Sorry, it's my fault. I forgot to upload the updated files. I submitted them this time.

toolstack commented:
Np, it happens 😉

PR looks good, I'll commit it and do a release in a day or two to see if anything else pops up that needs fixing.
